Microsoft.Toolkit.Uwp.SampleApp/SamplePages/FrameworkElementEx/FrameworkElementExCode.bind

```diff
@@ -0,0 +1,129 @@
+<Page xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:common="using:Microsoft.Toolkit.Uwp.SampleApp.Common"
+      xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:extensions="using:Microsoft.Toolkit.Uwp.UI.Extensions"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d">
+
+  <Grid x:Name="RootGrid" Background="{ThemeResource ApplicationPageBackgroundThemeBrush}" Padding="48">
+    <Grid.RowDefinitions>
+      <RowDefinition MinHeight="100" MaxHeight="500" />
+      <RowDefinition Height="11" />
+      <RowDefinition />
+    </Grid.RowDefinitions>
+    <Grid.ColumnDefinitions>
+      <ColumnDefinition MinWidth="100" MaxWidth="800" />
+      <ColumnDefinition />
+    </Grid.ColumnDefinitions>
+
+    <Rectangle Grid.Column="0"
+               Grid.Row="0"
+               extensions:FrameworkElementEx.EnableActualSizeBinding="@[EnableActualSizeBinding:Bool:True]"
+               Fill="{StaticResource Brush-White}"
+               Stroke="{StaticResource Brush-Grey-04}"
+               x:Name="TargetObject"
+               StrokeThickness="1"/>
+
+    <TextBlock Grid.Column="0"
+               TextWrapping="Wrap"
+               Margin="12,12">
+      Drag the handles to see how values are updated in the row below.
+      If EnableActualSizeBinding is set to false, the binding will not be updated.
+    </TextBlock>
+    <Grid Grid.Row="2"
+          Grid.Column="0"
+          Grid.ColumnSpan="2"
+          Margin="12,12">
+      <Grid.RowDefinitions>
+        <RowDefinition Height="Auto"/>
+        <RowDefinition Height="Auto"/>
+        <RowDefinition Height="Auto"/>
+        <RowDefinition Height="Auto"/>
+      </Grid.RowDefinitions>
+      <Grid.ColumnDefinitions>
+        <ColumnDefinition Width="*"/>
+        <ColumnDefinition Width="4*"/>
+      </Grid.ColumnDefinitions>
+      <TextBlock Text="ActualHeight:"
+                 Grid.Column="0"
+                 Grid.Row="0"/>
+      <TextBlock Text="{Binding ElementName=TargetObject, Path=ActualHeight}"
+                 Grid.Column="1"
+                 Grid.Row="0"/>
+
+      <TextBlock Text="ActualWidth:"
+                 Grid.Column="0"
+                 Grid.Row="1"/>
+      <TextBlock Text="{Binding ElementName=TargetObject, Path=ActualWidth}"
+                 Grid.Column="1"
+                 Grid.Row="1"/>
+
+      <TextBlock Text="BindableActualHeight:"
+                 Grid.Column="0"
+                 Grid.Row="2"/>
+      <TextBlock Text="{Binding ElementName=TargetObject, Path=(extensions:FrameworkElementEx.ActualHeight)}"
```

Comment:
It'd be a nice if we maybe split the demo into two sections and also bound these properties to a rectangle on the other side that was the same size as the rectangle in the grid.  Just to show a case that it might be more commonly used for.

Reply:
and allow users to scale a rectangle, so that another rectangle will automatically scale.

Reply:
Can you guys please elaborate on this? I'm not quite sure to understand what you mean.
The goal of the sample was just to show how the new property is updated while the current one doesn't, I didn't want to over-complicate things.

Reply:
@michael-hawker @Vijay-Nirmal

Reply:
In the real world, no one will bind ActualSize to a TextBlock. So if you bind that rectangle to another rectangle then it will somewhat stimulate the real world example. @ST-Apps What do you think?

Reply:
Well, this is a toolkit for developers tho. The goal of the samples app should be to show what's inside and how to use it, choosing what to bind to is up to the dev :)

Reply:
@ST-Apps right, it's true we want to show devs how to use the property.  But they're usually going to use it binding to some object to give it a specific size.  So, showing the sample app using it in the context that they might actually use it is going to be a lot more helpful than just outputting the value to a textbox.

The sample app is also a check/balance we can easily look at to make sure things are working (especially when the OS updates and such).  By having a more practical example, it's easier for someone else to tell looking at the sample 1) what it's supposed to do and 2) that it's behaving correctly.  Outputting a number to a textbox doesn't help either of those.

Reply:
@michael-hawker @Vijay-Nirmal 

What about something like this?

![Sample](https://i.imgur.com/UfOqNMz.png)